I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/episode1.rpy", line 1340, in script
    $ deletefiles()
  File "game/episode1.rpy", line 1340, in <module>
    $ deletefiles()
  File "game/screens.rpy", line 629, in deletefiles
    fn = __slotname(str(i+1), None, True)
NameError: global name '_m1_screens__slotname' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/episode1.rpy", line 1340, in script
    $ deletefiles()
  File "C:\Users\hazel\Documents\RenPy\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\hazel\Documents\RenPy\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/episode1.rpy", line 1340, in <module>
    $ deletefiles()
  File "game/screens.rpy", line 629, in deletefiles
    fn = __slotname(str(i+1), None, True)
NameError: global name '_m1_screens__slotname' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Rocktato 0.0
Sun Jan 17 21:34:21 2021
